aws/src/main/java/org/apache/iceberg/aws/s3/BaseS3File.java

```diff
@@ -19,20 +19,23 @@
 
 package org.apache.iceberg.aws.s3;
 
+import java.util.List;
 import org.apache.iceberg.aws.AwsProperties;
 import org.apache.iceberg.metrics.MetricsContext;
 import software.amazon.awssdk.http.HttpStatusCode;
 import software.amazon.awssdk.services.s3.S3Client;
 import software.amazon.awssdk.services.s3.model.HeadObjectRequest;
 import software.amazon.awssdk.services.s3.model.HeadObjectResponse;
 import software.amazon.awssdk.services.s3.model.S3Exception;
+import software.amazon.awssdk.services.s3.model.Tag;
 
 abstract class BaseS3File {
   private final S3Client client;
   private final S3URI uri;
   private final AwsProperties awsProperties;
   private HeadObjectResponse metadata;
   private final MetricsContext metrics;
+  private List<Tag> tags;
```

Comment:
private final, also should be named `writeTags`

Reply:
I have renamed to `writeTags`.

Reply:
I have renamed to `writeTags`.